acceptanceTests/tap_test.go

```diff
@@ -632,3 +638,26 @@ func TestTapDumpLogs(t *testing.T) {
 		return
 	}
 }
+
+func TestRestrictedMode(t *testing.T) {
+	namespace := "mizu-tests"
+
+	t.Log("creating permissions for restricted user")
+	if err := applyKubeFilesForTest(
+		t,
+		"minikube",
+		namespace,
+		"../cli/cmd/permissionFiles/permissions-ns-tap.yaml",
+		"../cli/cmd/permissionFiles/permissions-ns-ip-resolution-optional.yaml",
+	); err != nil {
+		t.Errorf("failed to create k8s permissions, %v", err)
+	}
+
+	t.Log("switching k8s context to user")
+	if err := switchKubeContextForTest(t, "user-with-restricted-access"); err != nil {
```

Comment:
don't we need to switch back to the no restricted access user at the cleanup of the test